setup.py

```diff
@@ -1,12 +1,40 @@
 #!/usr/bin/env python
 
-from setuptools import setup
+from setuptools import setup, find_packages
 
-INSTALL_REQUIRES = ["koji", "rpm", "requests", "PyYAML", "simplejson"]
 
-setup(name='alt-src',
-      description='Tool for uploading custom patches to centos source rpm repositories',
-      version='1.0',
-      url='https://github.com/release-engineering/alt-src.git',
-      install_requires=INSTALL_REQUIRES,
-      scripts=['alt-src'])
+def get_description():
+    return "Tool for uploading custom patches to centos source rpm repositories"
+
+
+def get_requirements():
+    with open("requirements.txt") as f:
+        return f.read().splitlines()
+
+
+setup(
+      name="alt-src",
+      version="0.2.0",
```

Comment:
from 1.0 to 0.2.0? was already released? seems odd to me.

Reply:
Yeah, I was unsure about this. @rajulkumar and I agreed on having x.y.z scheme and that, instead of expanding '1.0' to '1.0.0', '0.1.0' would be more appropriate, as it isn't yet released on PyPI. We thought the changes from this PR would then bump '0.1.0' to '0.2.0'.

Anyone else have thoughts/concerns/suggestions here?
@release-engineering/content-delivery

Reply:
Agree we had the discussion. However, looking back at other pubtools-* projects, the version was bumped only after it was released to PyPI, which is not the case here yet. Also, x value in x.y.z was bumped to 1 from 0 when the api was stable which is the case here as the same script is already being used i.e. it looks stable. So suggest to keep the previous version.